app/views/uploads/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= form_tag({:action => "upload_zip"}, :multipart => true) do %>
+
+  <div class="row mb-3 input-group">
+    <%= label :library_pick, "Library:", class: "col-sm-2 col-form-label" %>
+    <div class='col-sm-10 ps-0'>
+      <%= select "post", "library_pick", Library.all.map{ |lib| [lib.name, lib.id] }, {include_blank: false}, {class: "form-control form-select"} %>
```

Comment:
I think the `post` here is extraneous, but I'm happy to tidy that up later. There's a slightly less wordy, more Railsy way of doing this but I won't get into explaining it all here!

Reply:
I'm sure there is a cleaner way to handle this. I've only done a small amount of Rails so my knowledge in that area is far from complete. If you'd like to, I believe Github will also let you edit this branch since it's an open PR to make that change if I understand the UI correctly (never tried so can't confirm there).